scripts/import-cloudflare-certs-into-jdk.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+#
+# Imports the Cloudflare WARP Gateway Root CA certificate into all Java installations.
+# This is necessary because Cloudflare Zero Trust WARP acts as a man-in-the-middle proxy,
+# intercepting SSL connections with its own certificate. Without importing this certificate,
+# Gradle and other Java tools fail with SSL handshake errors when downloading dependencies.
+# This script automatically finds all JDKs, stops Gradle daemons, and imports the certificate.
+
+set -eu
+
+# Get the directory where this script lives
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly SCRIPT_DIR
+
+source "${SCRIPT_DIR}/shellUtils.sh"
+
+# Certificate is in the Expensidev repo (two levels up from scripts directory)
+EXPENSIDEV_DIR="$(dirname "$(dirname "${SCRIPT_DIR}")")"
+readonly EXPENSIDEV_DIR
+readonly CERT_FILE="${EXPENSIDEV_DIR}/config/ssl/cloudflare-ca.pem"
+
+# Global constants
+readonly CERT_ALIAS="cloudflare-gateway-root"
+readonly KEYSTORE_PASSWORD="changeit"
```

Comment:
```suggestion
readonly KEYSTORE_PASSWORD="${JAVA_KEYSTORE_PASSWORD:-changeit}"
```

Allows it to be overridden if not default